ProjForLabs/app/src/main/java/com/example/projforlabs/RvAdapter.java

```diff
@@ -38,19 +48,22 @@ public RvAdapter(Context context, List<Driver> drivers) {
 
     @Override
     public void onBindViewHolder(final DriverViewHolder driverViewHolder, final int i) {
-        Picasso.get().load(drivers.get(i).getPhoto()).into(driverViewHolder.avatar);
-        driverViewHolder.driverName.setText(drivers.get(i).getName());
-        driverViewHolder.carModel.setText(drivers.get(i).getCarModel());
-        driverViewHolder.rating.setText(drivers.get(i).getRating());
-        driverViewHolder.status.setText(drivers.get(i).getStatus());
-        driverViewHolder.layout.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                openItemDetails(i);
-            }
-        });
+            Picasso.get().load(drivers.get(i).getPhoto()).into(driverViewHolder.avatar);
+            driverViewHolder.driverName.setText(drivers.get(i).getName());
+            driverViewHolder.carModel.setText(drivers.get(i).getCarModel());
+            driverViewHolder.rating.setText(drivers.get(i).getRating());
+            driverViewHolder.status.setText(drivers.get(i).getStatus());
+            driverViewHolder.layout.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    openItemDetails(i);
+
```

Comment:
Please follow what code changes you commit. It is hard to review when there are so many unnecessary changes added.